packages/client/src/components/form/FormFieldGenerator.tsx

```diff
@@ -869,11 +914,47 @@ class FormSectionComponent extends React.Component<Props> {
     })
   }
 
+  getOnClickHandle = (
+    fields: IFormField[],
+    field: IButtonFormField,
+    setFieldValue: (fieldName: string, value: any) => void,
+    setFieldError: (fieldName: string, message: string) => void
+  ) => {
+    const trigger = fields.find((f) => f.name === field.options.trigger)
```

Comment:
Can this logic of finding of fields listening for this onClick be built into the "Button component"? 
`packages/client/src/components/form/Button.tsx` for instance.
We need to minimise the amount of code on this "top level" of FormFieldGenerator

Reply:
I first tried to do this in the button component, then reverted back because the `fields` array is not available in the generated input field component

Reply:
Can you supply it to the component as a prop? I don't think it's a bad idea for components to have access to all form fields if they need to

Reply:
Ok I can do it, as it makes the implementation lot easier